systems/Query/06-Runtime_View.md

```diff
@@ -0,0 +1,38 @@
+# Runtime View
+
+## RCPoller
+
+A happy path when updating happens.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant Blazegraph
+    participant RCPoller
+    participant Wikibase Repository
+    
+    RCPoller->>+Blazegraph: Query last updated timestamp
+    Blazegraph->>+RCPoller: Last updated time
+    RCPoller->>+Wikibase Repository: Poll API for RecentChanges since time
+    Wikibase Repository->>+RCPoller: RecentChanges
+    RCPoller->>+Wikibase Repository: Get RDF for Entities
+    Wikibase Repository->>+RCPoller: Entity RDF
+    RCPoller->>+Blazegraph: RDF Write Queries
+    RCPoller->>+Blazegraph: Update Timestamp
+    Blazegraph->>+RCPoller: Success
+```
+
+A common error case for infrequently edited wikis is when the storage timestamp in Blazegraph appears to be too old.
```

Comment:
This sentence and the linked task and the diagram make it seam like the problem still exists. But the bug is closed? Can this error still occur? If so, how to deal with it?

Reply:
This can still occur, and the task afaik still includes the needed steps to resolve this situation.